CHANGELOG.md

```diff
@@ -4,16 +4,48 @@
 `psql` with the `-X` flag to prevent any `.psqlrc` commands from
 accidentally triggering the load of a previous DB version.**
 
-## 1.5.0 (unreleased)
+## 1.5.0 (2019-10-31)
+
+This release adds major new features and bugfixes since the 1.4.2 release.
+We deem it moderate priority for upgrading.
+
+This release adds compression of hypertable chunks as a major new feature.
```

Comment:
"as a major new feature.  Multiple type-specific compression options are available in this release (including DeltaDelta with run-length-encoding for integers and timestamps; Gorilla compression for floats; dictionary-based compression for any data type, but specifically for low-cardinality datasets; and other LZ-based techniques).  Individual columns can be compressed with type-specific compression algorithms as Postgres' native row-based format are rolled up into columnar-like arrays on a per chunk basis.  The query planner then handles transparent decompression for compressed chunks at execution time."